Jenkinsfile

```diff
@@ -511,7 +512,17 @@ pipeline {
     }
   }
   post {
-    cleanup {
+        cleanup {
+            withAzureAuth(){
+                whenTrue(isUnix()) {
+                    sh label: "Checking and removing any Azure related resource groups",
+                        script: "for group in `az group list --query \"[?name | starts_with(@,'${AZURE_RESOURCE_GROUP_PREFIX})']\" --out json|jq .[].name`;do az group delete --name $group --no-wait --yes;done"
+                }
+                whenFalse(isUnix()) {
```

Comment:
@russcam I think the cleanup always runs on the parent node which seems to be linux (ubuntu) is there a situation where this may not be the case? If not, we could probably remove this windows portion

Reply:
I'm not super familiar with the Jenkins setup so will defer to @elastic/observablt-robots. If it helps, it doesn't matter from what node/host the cleanup task runs from, so long as the input parameters are correct.